codec-stomp/src/main/java/io/netty/handler/codec/stomp/StompSubframeDecoder.java

```diff
@@ -226,14 +231,13 @@ private static void skipNullCharacter(ByteBuf buffer) {
     }
 
     private static void skipControlCharacters(ByteBuf buffer) {
-        byte b;
-        for (;;) {
-            b = buffer.readByte();
-            if (b != StompConstants.CR && b != StompConstants.LF) {
-                buffer.readerIndex(buffer.readerIndex() - 1);
-                break;
-            }
+        int offset = buffer.forEachByte(ByteProcessor.FIND_NON_CRLF);
```

Comment:
let's revert this as its not really related to the fix

Reply:
Ok, create new one for this ))